Makefile

```diff
@@ -227,26 +229,11 @@ $(DMG): nim_status_client $(DMG_TOOL)
 
 	# if MACOS_CODESIGN_IDENT is not set then the outer and inner .app
 	# bundles are not signed
-	[ -z "$(MACOS_CODESIGN_IDENT)" ] || \
```

Comment:
It's easier to standardise on `[[` instead of having to remember the differences.

Reply:
What differences? You already use `ifeq` in the `Makefile`, why don't you like `ifdef`?

Reply:
The syntactic differences between `[` and `[[`. (`-a` and `-o` versus `&&` and `||` to combine conditions)

Reply:
Ah, that's what you meant. Yes, I agree, whenever you can `[[` should be used instead of `[`.